content/02.introduction.md

```diff
@@ -1,29 +1,25 @@
 
-Studies of life sciences preprints have primarily focused on the metadata associated with these articles; however, their textual content remains unexamined.
+Preprints have often faced skepticism within the life science community [@doi:10.1371/journal.pgen.1008565; @doi:10.1136/bmj.m3111; @doi:10.1007/s10393-018-1352-3; @doi:10.1073/pnas.1511912112].
+One particular problem focuses on the publication process and the time it takes for preprints to become published.
+Previous work found that between two-thirds to three-quarters of preprints are eventually published [@doi:10.7554/eLife.45133.001; @doi:10.1002/pra2.175].
+In respect to time, previous work estimated that preprints take an average of 160 days to become published [@doi:10.2139/ssrn.3455146], while preprints with a single version often take less time than preprints with multiple versions [@doi:10.2139/ssrn.3455146].
+Despite these discoveries a missing piece is understanding which part of a preprint changes during the peer review process.
 
-We sought to understand the language landscape of preprints by performing a linguistic analysis of the bioRxiv corpus.
-We examined textual differences between preprints and published literature by comparing the entire corpus of preprints with articles available in the open access PubMed Central repository.
-We also examined linguistic differences between preprints and their corresponding published pairs.
-Examining this shift will provide a unique opportunity to ascertain parts of the peer review and publishing process and how it impacts the scholarly literature.
-Neural-network derived document embeddings provide a useful space for determining the textual similarity of preprints, which enables us to extend this work beyond word frequencies.
-Examining articles with particularly close proximity in this space reveals unannotated preprint-publication pairs that earlier analyses could not consider.
-In this space a preprint's nearest neighbors are also more likely than distant articles to share an eventual publishing venue with the preprint itself.
-We provide a webserver that will displays neighboring journals and articles for any preprint on bioRxiv or medRxiv, which can help authors identify similar papers or suitable journals.
-Our linguistic analysis, the first of the bioRxiv corpus, reveals the impact of the life sciences publishing process, introduces a method to identify matching preprint-published article pairs, demonstrates that the text content of preprints is related to their eventual publication venue, and provides a more complete picture of the fraction of preprints that are eventually published.
+In this work we sought to understand the language landscape within the bioRxiv corpus and contextualize how this language changes under the guise of peer review.
+To our knowledge this is the first time anyone has conducted a linguistic analysis on this repository.

```

Comment:
I think Larry wanted more than the primacy argument for significance:

> Why is this being done? There needs to be a significance argument here. There are controversies about preprints — how will this work address them?

Are there any interesting/important reasons to study this?